pkg/jx/cmd/install.go

```diff
@@ -274,6 +275,7 @@ func NewCmdInstall(commonOpts *CommonOptions) *cobra.Command {
 	cmd.Flags().StringVarP(&options.Flags.Provider, "provider", "", "", "Cloud service providing the Kubernetes cluster.  Supported providers: "+KubernetesProviderOptions())
 
 	cmd.AddCommand(NewCmdInstallDependencies(commonOpts))
+	awsCreateVaultOptions(cmd, &options.AWSConfig)
```

Comment:
These adds quite a bunch of options to install command which are only used for AWS/EKS. Maybe we should move these out into something more specific `create cluster eks` command, or is some configuration which is created during  when the cluster is created.

Reply:
`jx create cluster eks` and `jx create cluster aws` should have these options as well, but I want to use terraform to create my cloud resources, so I will never use these commands. I suppose one could split up `jx install` into several commands as well instead of having the option `--provider`. But I'll wait for @jstrachan to pitch in before starting with such a change.